### PR TITLE
Added 478

### PR DESCRIPTION
**Pull request by georgenetu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12238,4 +12238,32 @@ Find F(10^8).
...
+
+For example, say we have three mixtures with ratios (3 : 0 : 2), (3 : 6 : 11) and (3 : 3 : 4). By mixing 10 units of the first, 20 units of the second and 30 units of the third, we get a new mixture with ratio (6 : 5 : 9), since:
+(10·3/5 + 20·3/20 + 30·3/10 : 10·0/5 + 20·6/20 + 30·3/10 : 10·2/5 + 20·11/20 + 30·4/10) = (18 : 15 : 27) = (6 : 5 : 9)
+
+However, with the same three mixtures, it is impossible to form the ratio (3 : 2 : 1), since the amount of B is always less than the amount of C.
+
+Let _n_ be a positive integer. Suppose that for every triple of integers (_a_, _b_, _c_) with 0 ≤ _a_, _b_, _c_ ≤ n and gcd(_a_, _b_, _c_) = 1, we have a mixture with ratio (_a_ : _b_ : _c_). Let M(_n_) be the set of all such mixtures.
+
+For example, M(2) contains the 19 mixtures with the following ratios:
+{(0 : 0 : 1), (0 : 1 : 0), (0 : 1 : 1), (0 : 1 : 2), (0 : 2 : 1),
```
Comment (imsky) [inline README.md]: can this be formatted as a table or at least a code-formatted block?

Comment (georgenetu) [inline README.md]: The entire entry or just the math part?

Comment (imsky) [inline README.md]: just the math part

Comment (georgenetu) [inline README.md]: Sure, I'll add another commit very soon.

**Comment by imsky** [comment]
thanks, looks good

**Pull request merged**